src/main/java/crimson/ResultEmailer.java

```diff
@@ -0,0 +1,48 @@
+package crimson;
+
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.JsonNode;
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import karmosin.ContinuousIntegrationJob;
+
+public class ResultEmailer{
+
+    private static final String YOUR_DOMAIN_NAME = "sandboxfa8ac3414aae4acd9eecd9a9bc3659fe.mailgun.org";
```

Comment:
let's set these to be assigned via constructor

Reply:
they are assigned in constructor now